chia/full_node/mempool_manager.py

```diff
@@ -550,8 +588,11 @@ def get_mempool_item(self, bundle_hash: bytes32, include_pending: bool = False)
         """
         item = self.mempool.spends.get(bundle_hash, None)
         if not item and include_pending:
-            # no async lock needed since we're not mutating the potential_cache
-            item = self.potential_cache._txs.get(bundle_hash, None)
+            # no async lock needed since we're not mutating the pending_cache
+            item = self._pending_cache.get(bundle_hash)
+        if not item and include_pending:
```

Comment:
May as well merge this condition with the one above:

`if not item and include_pending:`

Reply:
Oh, are these caches meant to be overlapping, or not? Better comment on that somewhere.

Reply:
Oh, I see. we first try pending, then conflicting.

Reply:
both `_conflict_cache` and `_pending_cache` are considered "pending" for the purposes of this API call. an item should only ever be in one or the other (or neither)